integrated_channels/degreed/exporters/content_metadata.py

```diff
@@ -37,10 +49,24 @@ def transform_description(self, content_metadata_item):
         We choose one value out of the course's full description, short description, and title
         depending on availability and length limits.
         """
-        full_description = content_metadata_item.get('full_description') or ''
-        if 0 < len(full_description) <= self.LONG_STRING_LIMIT:  # pylint: disable=len-as-condition
-            return full_description
-        return content_metadata_item.get('short_description') or content_metadata_item.get('title') or ''
+        course_runs = content_metadata_item.get('course_runs')
+        course_run_duration_info = get_course_run_duration_info(
+            get_closest_course_run(course_runs)
+        ) if course_runs else ''
+        description = add_duration_info(
```

Comment:
I thin we should not call `add_duration_info ` twice here. I think we can simplify logic here something like this
```
if 0 < len(full_description + course_run_duration_info) <= self.LONG_STRING_LIMIT:
    course_description = full_description
else:
   course_description = content_metadata_item.get('short_description') or content_metadata_item.get('title') or ''
return "{duration_info}{description}".format(duration_info=duration_info, description=course_description)
```
we probably don't need `add_duration_info` method